src/classes/class-opcache-stats.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Niteo\WooCart\Defaults\OpCacheStats {
+
+	use \WooCart\Log\Socket;
+
+	/**
+	 * Class Reporter
+	 *
+	 * @package Niteo\WooCart\Defaults\OpCacheStats
+	 */
+	class Reporter {
+
+		const cron_hook = 'WOOCart\OpCacheStats\Reporter';
+
+
+		public function __construct() {
+
+			if ( ! wp_next_scheduled( self::cron_hook ) ) {
+				wp_schedule_event( time(), 'hourly', self::cron_hook );
+			}
+
+			add_action( self::cron_hook, [ $this, 'report_stats' ] );
+		}
+
+		/**
+		 * Hook function that runs as part of cron.
+		 *
+		 * @return bool
+		 *
+		 * @codeCoverageIgnore
+		 */
+		public function report_stats(): bool {
+			$plugins = $this->parse_plugins( opcache_get_status() );
+			$plugins = $this->decorate( $plugins );
+			return $this->emit( $plugins );
+		}
+
+		/**
+		 * Parse opcache stats and sum stats by plugin.
+		 *
+		 * @param array $report
+		 * @return array
+		 */
+		public function parse_plugins( array $report ): array {
+			$scripts        = $report['scripts'];
+			$wp_plugin_path = wp_normalize_path( WP_PLUGIN_DIR );
+			$plugin_start   = strlen( $wp_plugin_path ) + 1;
+			$plugins        = [];
+			foreach ( $scripts as $script => $stats ) {
+				if ( ! strstr( $script, $wp_plugin_path ) ) {
+					continue;
+				}
+				$slug = substr( $script, $plugin_start, $plugin_start );
```

Comment:
This part is confusing. What happens if slug is longer than `$wp_plugin_path`?

Reply:
Can't be, line 51 verifies that $script has to contain wp_plugin_path.

Reply:
@am-on re-review, renamed things so it should make more sense. And added an additional test case.